app/scripts/lib/fxa-client.js

```diff
@@ -399,25 +403,31 @@ define(function (require, exports, module) {
         clientOptions.resume = options.resume;
       }
 
+      setMetricsContext(clientOptions, options);
+
       return client.passwordForgotResendCode(
         email,
         passwordForgotToken,
         clientOptions
       );
     }),
 
-    completePasswordReset: withClient((client, originalEmail, newPassword, token, code, relier) => {
+    completePasswordReset: withClient((client, originalEmail, newPassword, token, code, relier, options = {}) => {
       const email = trim(originalEmail);
 
-      return client.passwordForgotVerifyCode(code, token)
+      var clientOptions = {
+        keys: wantsKeys(relier),
+        sessionToken: true
+      };
+
+      setMetricsContext(clientOptions, options);
+
+      return client.passwordForgotVerifyCode(code, token, clientOptions)
```

Comment:
The `keys` and `sessionToken` parameters are the concerning ones. We are depending on `fxa-js-client` to filter those out, which should be the case, but no real need to depend on that?